src/main/resources/application.yml

```diff
@@ -9,12 +9,18 @@ spring:
     # in case you want to use H2 with a file persistence or as a service
     # reference http://www.h2database.com/html/cheatSheet.html and
     # http://www.h2database.com/html/tutorial.html#using_server
-    url: jdbc:h2:mem:testtest
+    url: jdbc:h2:mem:testdb
     driver-class-name: org.h2.Driver
+    username: sa
+    password:
   jpa:
     hibernate:
       ddl-auto: update
     database-platform: org.hibernate.dialect.MySQL5Dialect
+  h2:
```

Comment:
If you ever want to see what is actually being queried behind the scenes, you can use 
`spring.jpa.show-sql=true` and `spring.jpa.properties.hibernate.format_sql=true` to log out the queries.
https://www.baeldung.com/sql-logging-spring-boot
This should only be used for local testing though